backend/WP/Gutenberg/AbstractBlock.php

```diff
@@ -101,7 +101,7 @@ public function render( array $block, string $content, bool $is_preview, int $po
 				[
 					'block',
 					'block-' . $this->get_name(),
-					'has-text-align-' . $block['align'] ?? 'left',
+					'has-text-align-' . ( '' === $block['align'] ? 'left' : $block['align'] ),
```

Comment:
I think this does exactly the same as the removed line, but longer. We could keep the previous version of this (unless you think it doesn't work).

Reply:
I think this was added for cases when $block['align'] exists but is an empty string, since the ?? operator only checks for NULL